src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor().name();
+        int number = random.nextInt(100);
```

Comment:
Consider defining a constant for the maximum ball number (100). Using a named constant instead of a 'magic number' makes the code more readable and maintainable.